edgepi-leds.html

```diff
@@ -1,167 +1,188 @@
 <script type="text/javascript">
-    RED.nodes.registerType('led',{
-        category: 'EdgePi',
-        color: '#f391aa',
-        defaults: {
-            name: { value:"" },
-            transport: { value: "Local" },
-            config: { value: "Editor" },
-            method: { value: "toggleLed"},
-            ledPin: { value: "LED1" },
-            tcpAddress: { value: "" },
-            tcpPort: { value: "" }
-        },
-        inputs:1,
-        outputs:1,
-        icon: "light.svg",
-        label: function() {
-            return this.name||"led";
-        },
-        oneditprepare: function(){
-            // Query html elements
-            
-            const configTransportSelect = document.getElementById("node-input-transport");
-            const tcpField = document.querySelector(".form-row.tcp")
-            const configSelect = document.getElementById("node-input-config");
-            const methodField = document.querySelector(".form-row.method");
-            const ledPinField = document.querySelector(".form-row.ledPin");
+  RED.nodes.registerType("led", {
+    category: "EdgePi",
+    color: "#f391aa",
+    defaults: {
+      name: { value: "" },
+      transport: { value: "Local" },
+      ledState: { value: true },
+      ledPin: { value: 1 },
+      tcpAddress: { value: "" },
+      tcpPort: { value: "" },
+    },
+    inputs: 1,
+    outputs: 1,
+    icon: "light.svg",
+    label: function () {
+      return this.name || "led";
+    },
+    oneditprepare: function () {
+      const configTransportSelect = document.getElementById(
+        "node-input-transport"
+      );
+      const tcpField = document.querySelector(".form-row.tcp");
```

Comment:
don't name this tcpField! it is called transport!

Reply:
Updated